components/DocumentsList/index.js

```diff
@@ -64,7 +64,7 @@ const Document = ({ expandList, documents }) => (
     ) : (
       <div className="row">
         {documents.map(({ title, file }) => (
-          <BoxContainerCol key={file.key} className="col-12 col-sm-6 col-md-4 col-lg-3 col-xl-auto">
+          <BoxContainerCol key={file.url} className="col-12 col-sm-6 col-md-4 col-lg-3 col-xl-auto">
```

Comment:
Good catch 👍

Reply:
Well, React was shouting at me about it, so not so hard to catch. :)

Reply:
I saw it but I was assuming that it is because of the repeating same urls from contentful.